pctasks/dataset/pctasks/dataset/utils.py

```diff
@@ -28,27 +28,27 @@ def opt_collection(fn: Callable[..., Any]) -> Callable[..., Any]:
             "Collection ID to process. "
             "Must be supplied if multiple collections exist in the configuration."
         ),
-    )
+    )  # type: ignore[var-annotated]
     _opt(fn)
     return fn
 
 
 def opt_upsert(fn: Callable[..., Any]) -> Callable[..., Any]:
-    _opt = click.option("-u", "--upsert", is_flag=True, help="Upsert the workflow.")
+    _opt = click.option("-u", "--upsert", is_flag=True, help="Upsert the workflow.")  # type: ignore[var-annotated]  # noqa: E501
     _opt(fn)
     return fn
 
 
 def opt_submit(fn: Callable[..., Any]) -> Callable[..., Any]:
-    _opt = click.option("-s", "--submit", is_flag=True, help="Submit the workflow.")
+    _opt = click.option("-u", "--upsert", is_flag=True, help="Upsert the workflow.")  # type: ignore[var-annotated]  # noqa: E501
```

Comment:
I blame my editor for this accidental change.

```suggestion
    _opt = click.option("-s", "--submit", is_flag=True, help="Submit the workflow.")  # type: ignore[var-annotated]  # noqa: E501
```